app/styles/paper-toast-extras.scss

```diff
@@ -0,0 +1,6 @@
+// Ember-paper extras for paper-toast
```

Comment:
There should be no styles within `ember-paper`. They should all come from `angular-source-material`. After you fix the import issue, hopefully you can just remove these and the angular styles will match your markup.